docs/source/changes.md

```diff
@@ -120,3 +120,7 @@ authors:
 - `epochs_tmin` and `epochs_tmax` were named incorrectly in some test config
   files.
   ({{ gh(340) }} by {{ authors.crsegerie }})
+- We now reject bad epochs by using `ica_reject` before plotting the 
```

Comment:
Can you insert a link to the docs here? 
```
[`ica_reject`][config....]
```

Reply:
Proposal:

"… before producing the "overlay" plots that show the evoked data before and after ICA cleaning."

Reply:
ok!